<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MYDBSMClass</class>
 <widget class="QMainWindow" name="MYDBSMClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>807</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MYDBSM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前数据库路径：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_choice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_root">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>控制台：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_command">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>1321</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_commit">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>720</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>1321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1319</width>
       <height>179</height>
      </rect>
     </property>
     <widget class="QListWidget" name="list_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table_result2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前展示表格：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_table">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1359</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MYDBSM.qrc"/>
 </resources>
 <connections/>
</ui>
